files/ru/web/javascript/reference/classes/public_class_fields/index.html

```diff
@@ -1,25 +1,51 @@
 ---
-title: Поля классов
+title: Публичные поля классов
 slug: Web/JavaScript/Reference/Classes/Public_class_fields
 tags:
-  - JavaScript
   - Классы
+  - JavaScript
+  - Возможности языка
 translation_of: Web/JavaScript/Reference/Classes/Public_class_fields
 original_slug: Web/JavaScript/Reference/Classes/Class_fields
 ---
 <div>{{JsSidebar("Classes")}}</div>
 
-<div class="note">Объявление публичных и приватных полей является <a href="https://github.com/tc39/proposal-class-fields">экспериментальной разработкой</a>, предложенной в коммитете стандартов JavaScript <a href="https://tc39.github.io/beta/">TC39</a>. Поддержка браузерами ограничена, но данное нововведение можно использовать посредством транспиляторов (например, <a href="https://babeljs.io/">Babel</a>). Смотрите информацию о совместимости ниже.</div>
+<div class="note">
+  <p><strong>Примечание:</strong> Это страница описывает экспериментальные возможности.</p>
 
-<h2 id="Публичные_поля">Публичные поля</h2>
+  <p>Публичные и приватные поля — это <a
+      href="https://github.com/tc39/proposal-class-fields">экспериментальная функция (stage
+      3)</a>, предложенная комитетом по стандарту JavaScript <a href="https://tc39.es/">TC39</a>.</p>
+
+  <p>Поддержка этой возможности в браузерах ограничена, но ее можно использовать по средством транспилирования, с такими системами как <a href="https://babeljs.io/">Babel</a>. Смотрите <a href="#browser_compatibility">информацию о совместимости</a> ниже.</p>
```

Comment:
```suggestion
  <p>Поддержка этой возможности в браузерах ограничена, но ее можно использовать посредством транспилирования с такими системами как <a href="https://babeljs.io/">Babel</a>. Смотрите <a href="#browser_compatibility">информацию о совместимости</a> ниже.</p>
```

Reply:
https://ru.wiktionary.org/wiki/%D0%BF%D0%BE%D1%81%D1%80%D0%B5%D0%B4%D1%81%D1%82%D0%B2%D0%BE%D0%BC